src/http/api/views/account.posts.view.ts

```diff
@@ -82,8 +82,8 @@ export class AccountPostsView {
 
     async getPostsByApId(
         apId: URL,
-        account: PersistedAccount,
-        currentContextAccount: PersistedAccount,
+        account: Account | null,
```

Comment:
No more PersistedAccount! But this _is_ nullable, I needed to fix the type. We do `account?.isInternal` below - so this is safe